src/ActiveBackground/Confetti/EulerMass.ts

```diff
@@ -1,37 +1,31 @@
 import { Vector2D } from './Vector'
 
-class EulerMass {
-  position: Vector2D
-  mass: number
-  drag: number
-  force: Vector2D
-  velocity: Vector2D
+export class EulerMass {
+  public position: Vector2D
+  private force: Vector2D
+  private velocity: Vector2D
 
-  constructor(x: number, y: number, mass: number, drag: number) {
+  constructor(x: number, y: number, private readonly mass: number, private readonly  drag: number) {
```

Comment:
You can inline properties if you which. I personally like it. Some people don't.

Reply:
I've never seen that before in typescript, I like it, It looks good to me